getting-started/try-key-features-timescale-products.md

```diff
@@ -0,0 +1,430 @@
+$CONSOLE. You can also do this using psql. 
+
+<Tabs label="Upload data to ">
+
+<Tab title="SQL Editor">
+
+<Procedure>
+
+1. **In [$CONSOLE][portal-ops-mode], select the service you uploaded data to, then click `SQL Editor`**
+
+1.  **Create a continuous aggregate**
+
+    For a continuous aggregate, data grouped using a time bucket is stored in a
+    PostgreSQL `MATERIALIZED VIEW` in a hypertable. `timescaledb.continuous` ensures that this data
+    is always up to date.
+    In your SQL editor, use the following code to create a continuous aggregate on the real-time data in
+    the `crypto_ticks` table:
+
+    ```sql
+    CREATE MATERIALIZED VIEW assets_candlestick_daily
+    WITH (timescaledb.continuous) AS
+    SELECT
+      time_bucket('1 day', "time") AS day,
+      symbol,
+      max(price) AS high,
+      first(price, time) AS open,
+      last(price, time) AS close,
+      min(price) AS low
+    FROM crypto_ticks srt
+    GROUP BY day, symbol;
+    ```
+
+    This continuous aggregate creates the [candlestick chart][charts] data you use to visualize
+    the price change of an asset.
+
+1. **Create a policy to refresh the view every hour**
+
+   ```sql
+   SELECT add_continuous_aggregate_policy('assets_candlestick_daily',
+   start_offset => INTERVAL '3 weeks',
+   end_offset => INTERVAL '24 hours',
+   schedule_interval => INTERVAL '3 hours');
+   ```
+
+1.  **Have a quick look at your data**
+
+    You query continuous aggregates exactly the same way as your other tables. To query the `assets_candlestick_daily`
+    continuous aggregate for all assets:
+
+    <TryItOutCodeBlock queryId="getting-started-crypto-cagg" />
+
+
+</Procedure>
+
+</Tab>
+
+<Tab title="Continuous aggregate wizard">
+
+<Procedure>
+
+1. **In [$CONSOLE][portal-ops-mode], select the service you uploaded data to**.
+1. **Click `Operations` > `Continuous aggregates`, select `crypto_ticks`, then click `Create continuous aggregate`**.
+   ![Continuous aggregate wizard](https://assets.timescale.com/docs/images/continuous-aggregate-wizard.png )
+1. **Create a view called `assets_candlestick_daily` on the `time` column with an interval of `1 day`, then click `Next step`**.
+1. **Update the view SQL with the following functions, then click `Run`**
+   ```sql
+   CREATE MATERIALIZED VIEW assets_candlestick_daily
+   WITH (timescaledb.continuous) AS
+   SELECT 
+     time_bucket('1 day', "time") AS bucket,
+     symbol,
+     max(price) AS high,
+     first(price, time) AS open,
+     last(price, time) AS close,
+     min(price) AS low
+   FROM "public"."crypto_ticks" srt
+   GROUP BY bucket, symbol;
+    ```
+1. **When the view is created, click `Next step`**
+1. **Define a refresh policy with the following values, then click `Next step`**
+   - `How far back do you want to materialize?`: `3 weeks`
+   - `What recent data to exclude?`: `24 hours`
+   - `How often do you want the job to run?`: `3 hours`
+1. **Click `Create continuous aggregate`, then click `Run`**
```

Comment:
You only click Run here, there is no create button.

Reply:
Fixed